src/sentry/static/sentry/app/components/charts/baseChart.tsx

```diff
@@ -489,6 +489,32 @@ const ChartContainer = styled('div')`
   .echarts-for-react div:first-of-type {
     width: 100% !important;
   }
+
+  /* Tooltip description styling */
+  .tooltip-description {
+    color: ${p => p.theme.white};
+    border-radius: ${p => p.theme.borderRadius};
+    background: #000;
```

Comment:
Should this vary based on the theme?

Reply:
unfortunately, we don't have this color in our theme and I would like to make it consistent with the color of our tooltip. The theme black color is ` #1D1127`